### PR TITLE
ORLP-217 fix demo 2 

### PR DESCRIPTION
**Pull request by Tamara20**

### CODE REVIEW

File: src/main/java/com/softserve/academy/spaced/repetition/controller/ExceptionHandlerController.java [hunk 1]
```diff
@@ -20,12 +20,12 @@
 @ControllerAdvice
 public class ExceptionHandlerController extends ResponseEntityExceptionHandler {
 
-    private static final EnumMap<AccountStatus, ResponseEntity<MessageDTO>> MAP = new EnumMap<AccountStatus, ResponseEntity<MessageDTO>>(AccountStatus.class);
+    private static final EnumMap<AccountStatus, ResponseEntity<MessageDTO>> STATUS = new EnumMap<>(AccountStatus.class);
```
Comment (antonbakalets) [inline src/main/java/com/softserve/academy/spaced/repetition/controller/ExceptionHandlerController.java]: I would rename to something like USER_STATUS_ERROR_RESPONSE.

File: src/main/java/com/softserve/academy/spaced/repetition/service/AccessToUrlService.java [hunk 2]
```diff
@@ -85,6 +85,12 @@ public boolean hasAccessToDeleteCommentForCourse(Long commentId) throws NotAutho
         return hasAccessToUpdateCommentForCourse(commentId) || hasRoleAdmin;
     }
 
+    public boolean hasAccessToUpdateCommentForCourse(Long commentId) throws NotAuthorisedUserException {
```
Comment (antonbakalets) [inline src/main/java/com/softserve/academy/spaced/repetition/service/AccessToUrlService.java]: What is the reason to move this method in this pull request? Maybe it worths revert this.

File: src/main/java/com/softserve/academy/spaced/repetition/controller/ExceptionHandlerController.java (hunk 1 above)
Comment (Tamara20) [reply src/main/java/com/softserve/academy/spaced/repetition/controller/ExceptionHandlerController.java]: ok) thank you

File: src/main/java/com/softserve/academy/spaced/repetition/service/AccessToUrlService.java (hunk 2 above)
Comment (Tamara20) [reply src/main/java/com/softserve/academy/spaced/repetition/service/AccessToUrlService.java]: yes I agree

File: src/main/resources/mailTemplates/deleteAccountMailTemplate.txt [hunk 3]
```diff
@@ -1,10 +0,0 @@
-<html>
```
Comment (antonbakalets) [inline src/main/resources/mailTemplates/deleteAccountMailTemplate.txt]: I hope you have a good reason to remove this template.

**Pull request merged**